pallets/runtime/src/asset.rs

```diff
@@ -176,6 +181,20 @@ pub enum TickerRegistrationStatus {
     RegisteredByDid,
 }
 
+/// The type of an identifier associated with a token.
```

Comment:
wrong comment I think

Reply:
Yeah, I think it is an outcome of copy-paste of the data structure.  I will change it...